### ISSUE TITLE
NavRoute: Investigate loading the data during 'catching_up'

### ISSUE THREAD

**Issue by Athanasius** [EDCD/EDMarketConnector#1739]
vStone on Discord has asked if we can please load NavRoute during 'catch up' on starting up EDMC.

The code currently explicitly does not do this when catching up... but there's every chance that is leftover from before we did the "retry and check the timestamps" thing to avoid loading an outdated file.  Also would have been before NavRouteClear, so we wouldn't have known if the NavRoute.json contents had since been cleared.

We can probably just remove the `and not self.catching_up` from https://github.com/EDCD/EDMarketConnector/blob/dd4a178a6729049c05d0c0a632f798722e536494/monitor.py#L1372 , but need to double-check doing so won't have undesirable side-effects.

**Comment by Athanasius** [comment]
Some notes:

1. The way that `__navroute_retry()` returns False if it can't load the file contents is to handle where writing actual (new) content to the file lags behind the event being in the Journal.  Thus if the NavRoute.json is in this state, which includes just not existing, then during catch up we would Keep On Trying To Read The File for every event.  I'd rather not making the logic in there any more convoluted if I can help it.  It'd have to be something like:
```python
if self.catching_up:
	self._last_navroute_journal_timestamp = None
```
before the 'return False'.
2. Best case we'll load it once for every NavRoute event in the latest Journal file.

**Comment by Athanasius** [comment]
There's an alternative, depending on how well it fits the current code.  Just add a call to `EDLogs._parse_navroute_file()` after catchup is completed.  You're going to be relying on the first 'live' journal event to even trigger `journal_entry()` anyway, and can just check `state['NavRoute']` then.  Remember to check the timestamp in that data if needs be, and that it's not a 'NavRouteClear' content.

**Comment by Athanasius** [comment]
Trying out that hopefully simpler alternative ... 

1. We can just load the file and populate `monitor.state['NavRoute']` with *whatever it contains*, which might be a `NavRouteClear`.  But that's still potentially useful to plugins.
2. In the scenario of "game running, restart EDMC" the later EDMC invocation will send a synthetic `StartUp` event to plugins after catch-up, and *that* invocation of plugin `journal_entry()` will have the NavRoute data in `state`.

| Game | EDMC | Result |
|-----------------:|:-------|:--------|
| Not running, but NavRoute.json contains old route | Started a-new | NavRoute stored in `monitor.state`.  No `StartUp` to plugins because game not running. | 
| Started | Was running | `Fileheader` processing will clear `state['NavRoute']`.  First plugin event is `LoadGame`, `state['NavRoute'] == None`. |
| Enter GMap, old route re-plots | Still running | `NavRoute` event to plugins contains the route again. |
| Still running, route plotted. | Re-started | plugins get `StartUp` event with the route data in `state['NavRoute']` |

**Comment by Athanasius** [comment]
Now in released 5.7.0.
